lib/virtfs/xfs.rb

```diff
@@ -1 +1,20 @@
 require "virtfs/xfs/version"
+require_relative "xfs/allocation_group"
+require_relative "xfs/bmap_btree_block"
+require_relative "xfs/bmap_btree_record"
+require_relative "xfs/bmap_btree_root_node"
+require_relative "xfs/directory"
+require_relative "xfs/directory2_data_header"
+require_relative "xfs/directory3_data_header"
+require_relative "xfs/directory_block_tail"
+require_relative "xfs/directory_data_header"
+require_relative "xfs/directory_entry"
+require_relative "xfs/inode"
+require_relative "xfs/inode_map"
+require_relative "xfs/short_form_directory_entry"
+require_relative "xfs/short_form_header"
+require_relative "xfs/superblock"
+
+require_relative "xfs/base"
+require_relative "xfs/file_class"
+require_relative "xfs/dir_class"
```

Comment:
@Fryguy I don't see how that can be, since one depends on the size of load path array and the other does not. I can see that the difference may  be negligible, and thus, not a factor.

From what I've seen, it's not the standard pattern for every gem. I've seen a number of gems that use require_relative, mostly newer ones. True, most gems don't use it, but that's probably because require_relative hasn't been around as long. Unless there's some downside in using require_relative, I don't see any reason not to use it here.